README.md

```diff
@@ -1 +1,15 @@
-# hw01
\ No newline at end of file
+# Computing for SS HW1 Bio
+
+## Background information
+My name is Jinhui Hu, 1st year grad student with psych major under MAPSS.
+I considered myself as a cellular backgrounded person, and I am planning to work in Dr.Hansel's lab: http://www.hansellab-uchicago.com/

```

Comment:
This link should be embedded in the text itself:
[Dr.Hansel's lab](http://www.hansellab-uchicago.com/)
And this is how you do it:
`[Dr.Hansel's lab](http://www.hansellab-uchicago.com/)`